seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -138,6 +152,52 @@ def handle(self, *args, **options):
 
         logger.info('DONE')
 
+    @staticmethod
+    def _update_pdos(session, project_guid, sample_ids):
+        airtable_samples = session.fetch_records(
```

Comment:
does an airtable sample represent something completely different than a seqr sample?

Reply:
Like at a philosophical level no, but the PMs use airtable to track tons of stuff thats not tracked in  seqr